JSDemos/Demos/Scheduler/VirtualScrolling/description.md

```diff
@@ -1 +1,5 @@
-TODO
\ No newline at end of file
+If a **Scheduler** is bound to a large data set, enable Virtual Scrolling to optimize the widget's performance. In this mode, the widget renders only visible appointments. When an appointment leaves the viewport, the **Scheduler** removes it from the DOM.
```

Comment:
We don't usually capitalize feature names.

```suggestion
If a **Scheduler** is bound to a large data set, enable virtual scrolling to optimize the widget's performance. In this mode, the widget renders only visible appointments. When an appointment leaves the viewport, the **Scheduler** removes it from the DOM.
```

Reply:
```suggestion
If a **Scheduler** is bound to a large data set, enable virtual vcrolling to optimize widget performance. In this mode, the widget renders only visible appointments. When an appointment leaves the viewport, the **Scheduler** removes it from the DOM.
```